lib/include.js

```diff
@@ -486,6 +492,37 @@ Inclusion.include = function (objects, include, options, cb) {
      * Handle inclusion of HasMany relation
      * @param callback
      */
+    function includeHasManySimple(callback) {
+      //Map for Indexing objects by their id for faster retrieval
+      var objIdMap = includeUtils.buildOneToOneIdentityMap(objs, relation.keyFrom);
+      // all ids of primary objects to use in query
+      var sourceIds = Object.keys(objIdMap);
```

Comment:
The source ids will be string[]. It will probably breaks for number/objectid keys. Can we keep the original id values?